00-HTML-CSS-basics/Ex01_I.html

```diff
@@ -0,0 +1,12 @@
+<html>
```

Comment:
There is any special tag to announce to the browser which version of html we're using?